presto-postgresql/src/test/java/io/prestosql/plugin/postgresql/TestPostgreSqlTypeMapping.java

```diff
@@ -291,11 +309,39 @@ private void testUnsupportedDataType(String databaseDataType)
                 Function.identity());
     }
 
+    public static DataType<String> jsonDataType()
+    {
+        return DataType.dataType(
+                "json",
+                JSON,
+                value -> format("JSON'%s'", value),
```

Comment:
```suggestion
                value -> format("JSON '%s'", value),
```

Reply:
done

Reply:
This actually applies to in a few places in your test queries. Also, by a convention, we always uppercase type name in type constructors like `JSON '....'` rather than `json '...'` (or `json'...'`.)